modules/ServerAPI/src/client/SaltBrowser.html

```diff
@@ -0,0 +1,143 @@
+    <td class='bv_abbrev'><%-abbrev%></td>
+    <td class='bv_molFormula'><%-molFormula%></td>
+    <td class='bv_molWeight'><%-molWeight%></td>
+    <td class='bv_charge'><%-charge%></td>
+</script>
+
+<script type="text/template" id="SaltSummaryTableView" xmlns="http://www.w3.org/1999/html">
+    <table class="table table-bordered table-striped table-hover" style="margin: 0px; padding: 0px; border: 0px; width: 100%; ">
+        <thead style="">
+        <tr>
+            <th style="width: 125px;">Abbreviation</th>
+            <th style="width: 125px;">Salt Name</th>
+            <th style="width: 125px;">Formula</th>
+            <th style="width: 125px;">Molecular Weight</th>
+            <th style="width: 125px;">Charge</th>
+        </tr>
+        </thead>
+        <tbody style="">
+
+        </tbody>
+    </table>
+</script>
+
+<script type="text/template" id="SaltBrowserView" xmlns="http://www.w3.org/1999/html">
+
+    <div class="bv_saltSearchController" style="padding-bottom: 0px;"></div>
+    <div class="" style=" border: 1px solid #d4d4d4; border-top: 0px; margin: 0px; padding: 0px; ">
+
+        <div class="bv_searchSaltsStatusIndicator" style="height: 280px;">
+            <table width="100%">
+                <thead>
+                <tr><td style="display: none;">placeholder</td></tr>
+                </thead>
+                <tbody>
+                <tr>
+                    <td style="text-align: center; width: 100%; align-content: center; height: 250px;">
+                        <h5 class="bv_noMatchingSaltsFoundMessage hide">No Matching Salts Found</h5>
+                        <h5 class="bv_moreSpecificSaltSearchNeeded hide">A more specific search term is required.</h5>
+                        <h5 class="bv_errorOccurredPerformingSearch hide">An error occurred while searching for the salt.</h5>
+
+                        <div class="bv_saltBrowserSearchInstructions">
+                            <i class="icon-search bv_doSearchIcon"></i>
+                            Salt Browser
+                        </div>
+                        <div class="hide bv_searchingSaltsMessage" style="margin: auto;">
+                            <h5>Searching for Salts containing text "<i class="bv_exptSearchTerm"></i>"</h5>
+                            <div class="progress progress-striped active " style="width: 90%; margin: auto;">
+                                <div class="bar" style="width: 100%; margin: auto;"></div>
+                            </div>
+                        </div>
+                    </td>
+                </tr>
+                </tbody>
+                </tbody>
+            </table>
+        </div>
+
+        <div class="bv_saltTableController hide" style="margin:0px; padding:10px; overflow: auto; overflow: auto;"></div>
+    </div>
+    <div class="well well-small bv_saltControllerContainer hide" style="margin-top: 10px;">
+        <div>
+            <!-- Delete and Edit are not currently implemented! -->
```

Comment:
Can we hide these buttons for now or comment them out?
If we did decide to roll this feature out before we complete Edit and Delete, it could still be useful but we wouldn't want to include unusable buttons.